semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -186,6 +189,61 @@ def to_markdown_attr(
             )
         )
 
+    def to_markdown_attribute_table(
```

Comment:
Were there any changes to code here, besides extracting it?

Reply:
No changes to this function, looking at the git blame I haven't touched it since April